Source/EventFlow.ReadStores.ElasticSearch/Extensions/EventFlowOptionsExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using EventFlow.Aggregates;
+using EventFlow.Configuration;
+using Nest;
+
+namespace EventFlow.ReadStores.ElasticSearch.Extensions
+{
+    public static class EventFlowOptionsExtensions
+    {
+        public static EventFlowOptions UseElasticSearchReadModel<TAggregate, TReadModel>(this EventFlowOptions eventFlowOptions)
+            where TAggregate : IAggregateRoot
+            where TReadModel : class, IEsReadModel, new()
+        {
+            eventFlowOptions.AddRegistration(new Registration<IReadModelStore<TAggregate>, EsReadModelStore<TAggregate, TReadModel>>());
+            return eventFlowOptions;
+        }
+
+        public static EventFlowOptions ConfigureElasticSearch(this EventFlowOptions eventFlowOptions, IEsConfiguration esConfiguration)
+        {
+            eventFlowOptions.AddRegistration(
+                new Registration<IElasticClient>(
+                    r => new ElasticClient(new ConnectionSettings(new Uri(esConfiguration.ConnectionString), "eventflow"))));
```

Comment:
@tabro Maybe create an overload that takes the `ConnectionSettings` as argument and make sure to call `ThrowOnElasticsearchServerExceptions()` on the one that doesn't.